server/GraknServer.java

```diff
@@ -101,35 +94,14 @@ private GraknServer(ServerCommand.Start command) throws IOException {
         initLoggerConfig();
     }
 
-    private Server rpcServer() {
-        assert Executors.isInitialised();
-        return NettyServerBuilder.forPort(command.port())
-                .executor(Executors.service())
-                .workerEventLoopGroup(Executors.network())
-                .bossEventLoopGroup(Executors.network())
-                .maxConnectionIdle(1, TimeUnit.HOURS) // TODO: why 1 hour?
-                .channelType(NioServerSocketChannel.class)
-                .addService(graknService)
-                .addService(migratorRPCService)
-                .build();
-    }
-
-    private void initLoggerConfig() {
-        java.util.logging.Logger.getLogger("io.grpc").setLevel(Level.SEVERE);
-    }
-
-    private int port() {
-        return command.port();
-    }
-
-    private Path dataDir() {
-        return command.dataDir();
-    }
-
-    private void configureAndVerifyDataDir() throws IOException {
+    private void configureAndVerifyDataDir() {
         if (!Files.isDirectory(this.command.dataDir())) {
             if (this.command.dataDir().equals(ServerDefaults.DATA_DIR)) {
-                Files.createDirectory(this.command.dataDir());
+                try {
+                    Files.createDirectory(this.command.dataDir());
```

Comment:
Making the `configureAndVerifyDataDir` throw a runtime exception, since failing to create the data directory is not a recoverable condition.